adapters/feature_flags.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from amplitude_experiment import Experiment
+from amplitude_experiment import User
+
+import settings
+
+experiment = Experiment.initialize_local(settings.AMPLITUDE_DEPLOYMENT_KEY)
+
+
+def is_feature_enabled(
+    feature_name: str,
+    *,
+    user_id: str | None = None,
+    device_id: str | None = None,
+) -> bool:
+    if not experiment.poller.is_running:
+        experiment.start()
+
+    if device_id is None and user_id is None:
+        user_id = "1xx"
```

Comment:
we need to provide at least one, but for some features it may not be needed so we use a default value just to appease amplitude